frontend/src/components/camp/CampPicasso.vue

```diff
@@ -3,47 +3,96 @@ Listing all event instances of a single period.
 -->
 
 <template>
-  <v-card>
-    <v-toolbar dense class="mb-3" color="blue-grey lighten-5">
-      <v-icon left>
-        mdi-account-group
-      </v-icon>
-      <v-toolbar-title class="overflow-visible">
-        Picasso
-      </v-toolbar-title>
-      <v-tabs v-model="tab" right
-              center-active background-color="blue-grey lighten-5">
-        <v-tab v-for="period in periods.items"
-               :key="period.id">
-          {{ period.description }}
-        </v-tab>
-      </v-tabs>
-    </v-toolbar>
-    <v-tabs-items v-model="tab">
-      <v-tab-item v-for="period in periods.items"
-                  :key="period.id">
-        <v-skeleton-loader v-if="events._meta.loading" class="ma-3"
-                           type="table-thead,table-row@6" />
-        <picasso v-else
-                 :camp="camp"
-                 :event-instances="period.event_instances().items"
-                 :start="new Date(Date.parse(period.start))"
-                 :end="new Date(Date.parse(period.end))" />
-      </v-tab-item>
-    </v-tabs-items>
-  </v-card>
+  <card-view>
+    <div v-for="period in periods.items"
```

Comment:
Brauchts dieses `div` mit `v-for`? So wie ich das verstehe erstellt das eine separate `v-app-bar` für jede Periode...

Reply:
Ja, das war nur, damit alles angezeigt wird (in der Entwicklung)

Reply:
Also die ganze `v-app-bar` mit search und profile wird hier ja sowieso nicht angezeigt, oder? Der Code kommt dann wieder weg?

Reply:
@usu nein, die wird auf Mobile nur in diesem View angezeigt. Ich glaube das gibt nicht Sinn, die die TopBar Komponente der Camps zu packen.